<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Desktop-22" failures="1" tests="2" name="Dashboard.Customer_Landing.TC_07_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" time="212.185" errors="0" timestamp="2025-07-01T15:08:00 IST" skipped="1">
  <testcase classname="Dashboard.Customer_Landing.TC_07_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" name="TC_07_Verify_the_timeline_for_SH_test_with_unconfirmed_results" time="0.390">
    <skipped/>
  </testcase> <!-- TC_07_Verify_the_timeline_for_SH_test_with_unconfirmed_results -->
  <system-out/>
  <testcase classname="Dashboard.Customer_Landing.TC_07_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results" name="TC_13_Verify_the_display_of_calendly_calendar_with_message_for_SH_tests" time="211.795">
    <failure message="Message mismatch expected [Please note that you have the option to reschedule your appointment up to] but found [Please note that you have the option to reschedule your appointment up to 24 hours prior to the scheduled consultation.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Message mismatch expected [Please note that you have the option to reschedule your appointment up to] but found [Please note that you have the option to reschedule your appointment up to 24 hours prior to the scheduled consultation.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Test.Dashboard.Actions_Class.Customer_Landing_Actions.verifyCalendlyCalendarWithMsg(Customer_Landing_Actions.java:719)
at Dashboard.Customer_Landing.TC_07_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results.TC_13_Verify_the_display_of_calendly_calendar_with_message_for_SH_tests(TC_07_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results.java:17)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:658)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:219)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:923)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- TC_13_Verify_the_display_of_calendly_calendar_with_message_for_SH_tests -->
  <system-out/>
</testsuite> <!-- Dashboard.Customer_Landing.TC_07_Verify_The_Timeline_For_SH_Test_With_Unconfirmed_Results -->
